src/wallet/feebumper.cpp

```diff
@@ -20,7 +20,7 @@
 // calculation, but we should be able to refactor after priority is removed).
 // NOTE: this requires that all inputs must be in mapWallet (eg the tx should
 // be IsAllFromMe).
-int64_t CalculateMaximumSignedTxSize(const CWallet *pWallet, const CTransaction &tx)
+int64_t CalculateMaximumSignedTxSize(const CTransaction &tx, const CWallet *pWallet)
```

Comment:
In commit "Restore CalculateMaximumSignedTxSize function signature":

Previous signature took a CWallet reference instead of a pointer. (Just an observation, feel free to leave it.)